codalab_service.py

```diff
@@ -316,7 +330,7 @@ def _get_parser():
                 default=False,
             )
             cmd.add_argument(
-                'images',
+                '--images',
```

Comment:
Doesn't this make it so you can't do `./codalab_service.py build frontend` anymore?

Reply:
Yes it does.
The problem is that we add this option to every command, and that means we can't add any other positional arguments to any other command.

I think I can make it such that it stay a positional argument for the `build` command and named argument for the other commands. Let me try that.